src/util/unicode.cpp

```diff
@@ -256,8 +256,14 @@ const char **narrow_argv(int argc, const wchar_t **argv_wide)
   return argv_narrow;
 }
 
-std::wstring utf8_to_utf16(const std::string& in)
+std::wstring utf8_to_utf16be(const std::string& in)
```

Comment:
It may be acceptable to spell out big_endian (and, below, little_endian). Don't make others decipher names.